app/views/home/index.rb

```diff
@@ -6,13 +6,13 @@ def page_title
 
     def js_slsb_ld
       {
-        "@context": "http://schema.org",
-        "@type": "WebSite",
-        "url": "http://www.europeana.eu/portal",
-        "potentialAction": {
-          "@type": "SearchAction",
-          "target": "http://www.europeana.eu/portal/search?q={q}",
-          "query-input": "required name=q"
+        context: "http://schema.org",
+        type: "WebSite",
+        url: "http://www.europeana.eu/portal",
+        potentialAction: {
+          type: "SearchAction",
+          target: root_url + "search?q={q}",
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.